index.html

```diff
@@ -8,88 +8,107 @@
+            <div class="heroParent">
+                <div class="heroChild gapMedium floatUp">
+                    <header class="row gapSmall heroHeader floatUp">
+                        <img src="images/web/logo.png" alt="logo" class="heroIcon">
+                        <h1 class="heroTitle">OpenHV</h1>
+                    </header>
+                    <h3 class="heroDescription wrap floatUp p1">
+                        <p>A <b>classic</b> real-time strategy game,</p>
+                        <p>with a few <b>modern</b> twists.</p>
+                        <br />
+                        <p>Completely free and <b>open source</b></p>
+                        <p>from code to pixels and sound.</p>
+                    </h3>
 
-                <div class="row gapMedium wrap floatUp">
-                    <a href="#download" class="buttonPrimary"><span class="material-symbols-outlined">
-                            download
-                        </span>Download</a>
-                    <a href="#community" class="buttonPrimary"><span class="material-symbols-outlined">
-                            arrow_forward
-                        </span>More</a>
+                    <div class="row gapMedium wrap floatUp">
+                        <a href="#download" class="buttonPrimary"><span class="material-symbols-outlined">
+                                download
+                            </span>Download</a>
+                        <a href="#community" class="buttonPrimary"><span class="material-symbols-outlined">
+                                arrow_forward
+                            </span>More</a>
+                    </div>
                 </div>
             </div>
+            <div class="wrap imageToSolid">
+                
+            </div>
         </section>
-        <section class="section">
-            <div class="showcaseContainer gapMedium rowCollapsible">
-                <div class="showcaseParent gapMedium">
+        <section class="wrap section background2To1">
+            <h2 class="h2 textCenter">An old vision brought to life</h2>
+            <h3 class="h3 headerSoft textCenter">with a <a class="link" target="_blank" href="https://www.openra.net/">modern engine</a> and features</h3>
+        </section>
+        <div class="overviewContainer">
+            <div class="solidToImage"></div>
+            <section class="overviewParent overviewLeft gapMedium">
+
+                <div class="showcaseChild overviewContent gapMedium">
                     <div class="gapSmall centerColumn column">
                         <h3 class="h2">Classic RTS</h3>
                         <p class="p2 showcaseDescription">
-                            Journey back to the golden age of real-time strategy in a modern engine with todays convience and features.
+                            Journey back to the golden age of real-time strategy in a modern engine with todays
+                            convience and features.

```

Comment:
Why do you do these line breaks in the middle of the sentence?

Reply:
My formatter does this to limit the line length so that you don't have to scroll a ton